developing-with-nats/receiving/README.md

```diff
@@ -4,7 +4,6 @@ In general, applications can receive messages asynchronously or synchronously. R
 
 Some languages, like Go or Java, provide synchronous and asynchronous APIs, while others may only support one type of subscription.
 
-In all cases, the process of subscribing involves having the client library tell the NATS system that an application is interested in a particular subject.
-
-Under the covers, the client library will assign a unique id to each subscription. This id is used as a closure when the server sends messages to a specific subscription. Each subscription gets a unique id, so if the same connection is used multiple times for the same subject, the server will send multiple copies of the same message. When an application is done with a subscription it unsubscribes which tells the server to stop sending messages.
+In all cases, the process of subscribing involves having the client library tell the NATS system that an application is interested in a particular subject. When an application is done with a subscription it unsubscribes telling the server to stop sending messages.
 
+If a connection has multiple subscriptions using identical or overlapping subjects (say `foo` and `>`), the same message will be sent to the client multiple times.
```

Comment:
I would add something like "A client will receive a message for each matching subscription so if a connection..."

Reply:
added